docs/dcp2_schema_evolution_specs.rst

```diff
@@ -0,0 +1,158 @@
+
+.. role:: raw-html(raw)
+   :format: html
+.. |nn| replace:: :raw-html:`<blockquote>`
+.. |ne| replace:: :raw-html:`</blockquote>`
+
+==============================
+DCP2 Metadata Schema Evolution
+==============================
+
+.. contents::
+
+Overview
+========
+
+The HCA metadata standard is developed in a transparent and open manner so that the whole HCA community can participate
+in the process. It follows a set of `design principles`_ that serve as a foundation for the metadata standard in order
+to ensure it meets the needs of HCA data contributors, consumers and DCP downstream components.
+
+The metadata schemas are versioned following semantic `versioning rules`_ that account for backwards compatibility.
+However, compatibility of the schemas does not account for impact on the schema consumers, especially if different
+versions of the same schemas must be maintained, indexed, displayed and/or interacted with in any form.
+
+The metadata is updated through PRs in the `metadata-schema repository`_. The specific details can be found in the
+documentation in the repository, but the overall flow can be described as follows:
+
+- A ticket is opened describing an issue
+- A member of the Metadata Team looks into the issue, there is a discussion and, if deemed important, a PR is created
+  against the schema
+- The PR is reviewed within a time frame specified by the type of change (Major/Minor/Patch). The review includes
+  all the DCP components and test metadata is generated for each PR.
+- Once the required metadata team reviewers approve the changes, the branch is merged into staging
```

Comment:
To unspecific. Both the test metadata and the schema PRs need to be approved for either of them to be merged.